pandas/io/pytables.py

```diff
@@ -2567,7 +2575,7 @@ def read_index_node(self, node, start=None, stop=None):
         name = None
 
         if 'name' in node._v_attrs:
-            name = node._v_attrs.name
+            name = _ensure_decoded(_ensure_python_type(node._v_attrs.name))
```

Comment:
I guess you can't just do this unconditionally, as the name could be something like `None` or the literal `10` (not a string). So it'll be something like

```python
name = node._v_attrs.name
if isinstance(name, compat.string_types):
    name = compat.text_type(name)
```